### PR TITLE
update text to v1.3.0 when the assert-id already is from v1.3.0, or update assert id to v1.3.0 when text is from v1.3.0

### PR DESCRIPTION
**Pull request by cconcolato**
cc @podborski

### CODE REVIEW

**Comment by rbouqueau** [comment]
LGTM

**Comment by DenizUgur** [comment]
No regression, good to go

```
❯ git checkout master
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
❯ python aomrules.py --spec av1isobmff
AOM specification: https://aomediacodec.github.io/av1-isobmff/ (v1.3.0)
Check source file: ../src/specs/av1_isobmff/av1_isobmff.cpp

--------------------------------------------------
 SUMMARY
--------------------------------------------------
44 from 127 rules implemented.
83 rules are not implemented yet. Take a look at av1isobmff_todos.json
❯ gh pr checkout 85
Switched to branch 'fix_mismatches_v120_v130'
Your branch is up to date with 'origin/fix_mismatches_v120_v130'.
Already up to date.
❯ python aomrules.py --spec av1isobmff
AOM specification: https://aomediacodec.github.io/av1-isobmff/ (v1.3.0)
Check source file: ../src/specs/av1_isobmff/av1_isobmff.cpp

--------------------------------------------------
 SUMMARY
--------------------------------------------------
44 from 127 rules implemented.
83 rules are not implemented yet. Take a look at av1isobmff_todos.json
```

File: src/specs/av1_isobmff/av1_isobmff.cpp [hunk 1]
```diff
@@ -621,7 +621,7 @@ const SpecDesc specAv1ISOBMFF = {
     { "Section 2.2.4\n"
       "The config field SHALL contain an AV1CodecConfigurationBox that applies to the samples associated with this "
       "sample entry.",
-      "assert-f875c695",
+      "assert-8d3f8e0c",
```
Comment (cconcolato) [inline src/specs/av1_isobmff/av1_isobmff.cpp:624]: Looking at the code for this rule, it seems to be implementing `assert-318390e9` and `assert-a249db05`
```
Box Type:  av1C
Container: AV1 Sample Entry ('av01')
Mandatory: Yes 
Quantity:  Exactly One 
```
The proper way to implement "The config field SHALL contain an AV1CodecConfigurationBox" would be to verify that the "config" does not contain anything else than an AV1CodecConfigurationBox. The part  "that applies to the samples associated with this sample entry" is harder to implement.

**Review by DenizUgur — APPROVED** [review]

File: src/specs/av1_isobmff/av1_isobmff.cpp (hunk 1 above)
Comment (rbouqueau) [reply src/specs/av1_isobmff/av1_isobmff.cpp:624]: Should we put that in a separate issue to keep track?

**Review by rbouqueau — APPROVED** [review]
LGTM. We need to store @cconcolato 's comment to not lose track of it.

**Comment by podborski** [comment]
LGTM

**Pull request merged**